scorecard/views.py

```diff
@@ -176,26 +177,37 @@ def get_context_data(self, *args, **kwargs):
                     .first()
                     .as_dict()
                 )
-        infrastructure_financial_year = "2019/2020"
-        infrastructure = (
-            Project.objects.prefetch_related(
-                "geography",
-                "expenditure__budget_phase",
-                "expenditure__financial_year",
-                "expenditure",
-            )
-            .filter(
-                geography__geo_code=self.geo_code,
-                expenditure__budget_phase__name="Budget year",
-                expenditure__financial_year__budget_year=infrastructure_financial_year,
+        if Project.objects.all().count() > 0:
```

Comment:
It looks like the reason I added this was because local didn't work when there were no capital projects, I don't think that will ever be the case in prod. Will remove this